app/javascript/controllers/navbar_controller.js

```diff
@@ -0,0 +1,18 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = [ "mobileNavPanel" ]
+  connect() {
+    // this.element.textContent = "Hello World!"
```

Comment:
Do you need this?

Reply:
No, I forgot to remove it. I cleaned it up, thanks. :+1: